.github/workflows/create_release.yml

```diff
@@ -168,3 +168,62 @@ jobs:
         files: |
           ${{ steps.archive.outputs.tarball }}
           ${{ steps.archive.outputs.shasum }}
+
+  # Build to WebAssembly using Emscripten.
+  build-wasm:
+    name: wasm
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+    - name: install ninja
+      run: sudo apt-get install ninja-build
+    - name: emsdk install
+      run: |
+        mkdir $HOME/emsdk
+        git clone --depth 1 https://github.com/emscripten-core/emsdk.git $HOME/emsdk
+        $HOME/emsdk/emsdk update-tags
+        $HOME/emsdk/emsdk install tot
+        $HOME/emsdk/emsdk activate tot
+    - name: update path
+      run:  echo "PATH=$PATH:$HOME/emsdk" >> $GITHUB_ENV
+
+    # Configure with wasm EH and pthreads for maximal performance.
```

Comment:
I did mention that in the README docs added in this PR. It is Node 18.

Interesting idea about putting it in the name... but I think that might seem odd, as it's 18+. That is, I worry someone might have node 22 and think they need another build.

Reply:
How about at least calling to `-node` then.. since it can't run on anything else.

Reply:
Calling it just `-wasm` is little misleading as it might suggest that all you need is a wasm engine to run it.

Reply:
Well, it should actually run on deno and bun as well - they are all working to support node APIs now. I didn't test them though. Also, this build is not actually done with `ENVIRONMENT=node`, which means it can run on the web too

I get what you're saying about "wasm" being inaccurate but I feel "node" is inaccurate as well. These are basically generic js+wasm builds, or as generic as I can make them. I feel "wasm" is the more forward looking name as they are 95% wasm, and may some day become 100%.

Reply:
But the intended audience for this build is purely command line folks who will need to use node to run it right?

If somebody wants to run binaryen on the web then binaryen.js is a much better choice, right?

I suggest we *do* build with `-sENVIRONMENT=node` to make this intent clear (at least for now).

Reply:
Interesting. I hadn't thought of it that way.

I'm not sure `ENVIRONMENT=node` helps though as this isn't an assertions build, so there won't be a nice clear error message in another environment. I'd also prefer not to limit experimentation here, as experimentation is really the goal - maybe we'll find uses in the browser. E.g. the `binaryen.js` build uses the JS API, while this build uses the commandline API - maybe replicating commandline workflows would be simpler with it. 

But the more I think on it the more I agree with your larger point about the name, so I renamed it to `node` now and added some docs to clarify the purpose and status. What do you think?

Reply:
I get where you are coming from too.  Perhaps we can change this in the future but I like the its explicit for now.